cookbooks/aws-parallelcluster-platform/resources/c_states/c_states_redhat9.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
HEre and elsewhere: on new files the cipyright year should be 2024.

Reply:
Done